class None_User_Exception(Exception):
    pass
class User_Not_Found(Exception):
    pass